pkg/operator/operator.go

```diff
@@ -183,10 +183,11 @@ func NewOperator() (context.Context, *Operator) {
 	lo.Must0(mgr.AddHealthzCheck("healthz", healthz.Ping))
 	lo.Must0(mgr.AddReadyzCheck("readyz", healthz.Ping))
 
+	ctx = events.ToContext(ctx,events.NewRecorder(mgr.GetEventRecorderFor(appName)))
```

Comment:
Nice 🎉

Reply:
here there is problem , we have to pass the recorder to the baseContext but recorder depends on mgr which comes after baseContext